<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterRouteSheetConfigWidget</class>
 <widget class="QWidget" name="MasterRouteSheetConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>897</width>
        <height>830</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="organizationLabel">
         <property name="text">
          <string>Organization Key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="organizationLineEdit">
         <property name="text">
          <string>SEATTLE</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dateFormatlLabel">
         <property name="text">
          <string>Date Format String</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="dateFormatLineEdit">
         <property name="text">
          <string>d-MMM-yyyy</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="driverOffsetLabel">
         <property name="text">
          <string>Driver Cell Offset From Route Key : 0 = No Drivers</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QSpinBox" name="driverOffsetSpinBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="truckOffsetLabel">
         <property name="text">
          <string>Truck Cell Offset From Route Key : 0 = No Trucks</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QSpinBox" name="truckOffsetSpinBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="trailerOffsetLabel">
         <property name="text">
          <string>Trailer Cell Offset From Route Key : 0 = No Trailers</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QSpinBox" name="trailerOffsetSpinBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="clientIDLabel">
         <property name="text">
          <string>Client ID</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLineEdit" name="clientIDLineEdit">
         <property name="text">
          <string>Client ID</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="authURILabel">
         <property name="text">
          <string>Auth URI</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLineEdit" name="authURILineEdit">
         <property name="text">
          <string>Auth URI</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="x509Label">
         <property name="text">
          <string>Auth Provider x509 Cert URL</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QLineEdit" name="x509LineEdit">
         <property name="text">
          <string>x509 Provider</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="projectIDLabel">
         <property name="text">
          <string>Project ID</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="2">
        <widget class="QLineEdit" name="projectIDLineEdit">
         <property name="text">
          <string>Project ID</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="clientSecretLabel">
         <property name="text">
          <string>Client Secret</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="2">
        <widget class="QLineEdit" name="clientSecretLineEdit">
         <property name="text">
          <string>Client Secret</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="redirectURILabel">
         <property name="text">
          <string>Redirect URIs</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0" rowspan="2">
        <widget class="QListWidget" name="redirectURIListWidget"/>
       </item>
       <item row="21" column="1">
        <widget class="QPushButton" name="addRedirectURIPushButton">
         <property name="text">
          <string>Add URI</string>
         </property>
        </widget>
       </item>
       <item row="22" column="1">
        <widget class="QPushButton" name="removeSelectedRedirectURIPushButton">
         <property name="text">
          <string>Remove Selected URI</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="googleSheetsBaseAddrLabel">
         <property name="text">
          <string>Google Sheets Base Address</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0" colspan="2">
        <widget class="QLineEdit" name="googleSheetsBaseAddrLineEdit">
         <property name="text">
          <string>e.g. https://sheets.googleapis.com/v4/spreadsheets/&lt;someString&gt;/values/</string>
         </property>
        </widget>
       </item>
       <item row="25" column="0">
        <widget class="QLabel" name="googleSheetsScopeLabel">
         <property name="text">
          <string>Google Sheets API Scope</string>
         </property>
        </widget>
       </item>
       <item row="26" column="0" colspan="2">
        <widget class="QLineEdit" name="googleSheetsScopeLineEdit">
         <property name="text">
          <string>https://www.googleapis.com/auth/spreadsheets.readonly</string>
         </property>
        </widget>
       </item>
       <item row="27" column="0" colspan="2">
        <widget class="QLabel" name="requestTimeoutLabel">
         <property name="text">
          <string>Request Timeout (sec)</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0" colspan="2">
        <widget class="QSpinBox" name="requestTimeoutSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9000</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QLabel" name="oauth2UserTimerout">
         <property name="text">
          <string>OAuth2 User Authentication Timeout (sec)</string>
         </property>
        </widget>
       </item>
       <item row="30" column="0" colspan="2">
        <widget class="QSpinBox" name="oauth2UserTimeoutSpinBox">
         <property name="minimum">
          <number>40</number>
         </property>
         <property name="maximum">
          <number>9001</number>
         </property>
         <property name="value">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="31" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="32" column="0" colspan="2">
        <widget class="QPushButton" name="loadFromFilePushButton">
         <property name="text">
          <string>Load Settings From File</string>
         </property>
        </widget>
       </item>
       <item row="33" column="0" colspan="2">
        <widget class="QPushButton" name="saveSettingsPushButton">
         <property name="text">
          <string>Save Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>organizationLineEdit</tabstop>
  <tabstop>dateFormatLineEdit</tabstop>
  <tabstop>driverOffsetSpinBox</tabstop>
  <tabstop>truckOffsetSpinBox</tabstop>
  <tabstop>trailerOffsetSpinBox</tabstop>
  <tabstop>clientIDLineEdit</tabstop>
  <tabstop>authURILineEdit</tabstop>
  <tabstop>x509LineEdit</tabstop>
  <tabstop>projectIDLineEdit</tabstop>
  <tabstop>clientSecretLineEdit</tabstop>
  <tabstop>redirectURIListWidget</tabstop>
  <tabstop>addRedirectURIPushButton</tabstop>
  <tabstop>removeSelectedRedirectURIPushButton</tabstop>
  <tabstop>googleSheetsBaseAddrLineEdit</tabstop>
  <tabstop>googleSheetsScopeLineEdit</tabstop>
  <tabstop>requestTimeoutSpinBox</tabstop>
  <tabstop>oauth2UserTimeoutSpinBox</tabstop>
  <tabstop>loadFromFilePushButton</tabstop>
  <tabstop>saveSettingsPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
